lib/api_responder.rb

```diff
@@ -3,13 +3,28 @@ class APIResponder < ActionController::Responder
 
   def display(resource, given_options = {})
```

Comment:
Maybe we should rename this class to TwilioAPIResponder since now it has TwilioAPI specific details

Reply:
Yep, it makes sense.